### PR TITLE
Test on GPU

### PR DESCRIPTION
**Pull request by akawashiro**
Test applications using GPU through CUDA. These tests are assumed to fail for now.

### CODE REVIEW

**Comment by xuzijian629** [comment]
I'll introduce .pfnci tests in another PR soon.
You can run these tests in it

**Comment by akawashiro** [comment]
Thank you. 🙇

**Comment by akawashiro** [comment]
I will fix pfn-public-ci/sold.gpu in the two later PR.

File: libtorch_test/CMakeLists.txt [hunk 1]
```diff
@@ -27,13 +23,15 @@ add_custom_target(
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 
 add_custom_target(
-  torch_test_soldout_wo_section
-  EXCLUDE_FROM_ALL
+  torch_test_soldout_wo_section ALL
+  # EXCLUDE_FROM_ALL
```
Comment (xuzijian629) [inline libtorch_test/CMakeLists.txt]: Did you intentinoally include this to ALL?

Comment (akawashiro) [reply libtorch_test/CMakeLists.txt]: Ooops, just a mistake. Fixed in https://github.com/pfnet/sold/pull/2/commits/c826b7f7cdb2eca5d3e0abf6c4ea7a94e3d5c91b.

**Review by xuzijian629 — APPROVED** [review]
Thank you!

**Pull request merged**